### ISSUE TITLE
Add support for Firefox forks

### ISSUE THREAD

**Issue by plsng** [mkiol/GNotifier#47]
Would it be possible to add support for Firefox forks (IceWeasel, IceCat, ...)? By this I mean that the addon already works on them, but the notification displays a Firefox icon rather than the icon of the browser that's actually being used. Or is this already supposed to happen?

**Comment by kabili207** [comment]
There is a workaround for this already. In the add-on options called 'Notification icon' which _should_ be set to 'default' by default. The current code has checks for Iceweasel, Thunderbird, Firefox, and SeaMonkey with those exact spellings. If the [system.name](https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/system) doesn't match any of those values, it defaults to firefox.

There's also a distinct possibility that a particular fork doesn't change the system.name, in which case we'll need to find another way to determine which icon to use.

**Comment by mkiol** [comment]
That's right. For instance, if you run IceWeasel, Gnotifier will try to use "iceweasel" icon form your system (typically icon file is somewhere in /usr/share/icons/). If you want another icon, you can define different name on the add-on's settings page. 

The thing I can think about is to pack icon files (for Iceweasel, Icecat ect.) directly in to add-on package. 
Pros -> Icons will be displayed correctly regardless if file is present in /usr/share/icons/ or not.
Cons -> If system theme provides modified icon, it will not be used for notification.
